### ISSUE TITLE
Implement Alternate Sourceless Strategy

### ISSUE THREAD

**Issue by nakedsushi** [SCPR/deprecated-StreamMachine#3]
A continuation of #2 

I think, looking at `stream.coffee` that there already is support for an alternate source to fall back to.

AC:
- [x] replicate a scenario for when a fallback stream is triggered
- [x] create a silence mp3 for the fallback stream
- [x] see if the fallback stream works when the main source is not reachable by streammachine

**Comment by nakedsushi** [comment]
This makes it sound like there already is a fallback strategy:
https://github.com/StreamMachine/StreamMachine/issues/73

Test out:
- [x] having a fallback stream that's playing something else Nickleback??
- [x] pointing a silence stream to the fallback stream

**Comment by nakedsushi** [comment]
I just tested it out and it *is* currently possible to have a fallback stream with the current StreamMachine code. To make it work, I did this:

1. start streammachine standalone mode `./streammachine-cmd --config ./config/standalone.json ` with standalone.json looking something like this (notice the `fallback` option)
```
{
	"port":         8001,
	"source_port":  8002,
  "api_port":     8003,
	"log": {
		"stdout":     true
  },
  "streams": {
    "test": {
      "key":              "test",
      "root_route":       true,
      "format":           "mp3",
      "seconds":          3600,
      "metaTitle":        "Testing",
      "host":             "localhost",
      "source_password":  "testing",
      "fallback": "http://live.scpr.org/kpcclive/"
    }
  }
}
```
The fallback option above is just pointing to our live stream, but this can also be a stream of a silence mp3.

2. I started a "real" stream with liquidsoap, streaming my test playlist (pretending that this playlist was the real stream) to the now running streammachine instance: `liquidsoap 'output.icecast(%mp3, host="localhost", port=8002, password="testing", mount="test", mksafe(playlist("~/playlist.m3u")))'`

3. I then listened to the stream `http://127.0.0.1:8001/test` using VLC.  It started playing the audio from `playlist.m3u` as expected. I then killed the liquidsoap instance I had running from step 2, effectively killing the stream. After a second or two, VLC started playing audio from the livestream (my fallback option)

In theory, if we had another stream coming from somewhere, we can mount it on the same stream our livestream is on, mark it as a fallback, and streammachine should be able to switch to that when the primary stream is down.  I think that's what the fallback option Eric coded in was created for.
